usdevs/uscwebsite-hackathon-backend
# Add db seed

Closes #22. Added seed.ts to prisma to test out database seeding.

To test:
1. `docker-compose up -d` to start the development db.
2. `npm run prisma:migrate` to run the migrations
3. `npm run prisma:seed` to run the seeding.
4. Open the database to check if the rows are created.
![image](https://user-images.githubusercontent.com/44896529/207220466-b02a781e-2f59-41dd-9f1c-cd5bb047c5e4.png)
